puppet/modules/web/manifests/vhost/web.pp

```diff
@@ -27,7 +27,7 @@
     { 'rewrite_rule' => ['^/events/all.ics https://community.theforeman.org/c/events/l/calendar.ics [R,L]'] },
     { 'rewrite_rule' => ['^/events/? https://community.theforeman.org/c/events/l/calendar [R,L]'] },
     { 'rewrite_rule' => ["^/api/latest(.*) https://apidocs.theforeman.org/foreman/${stable}\$1 [R,L]"] },
```

Comment:
It's also here where I copied that blindly from, isn't this also incorrect?

Reply:
Ah now I see it, okay. Single quotes are one of the worst features in many languages, including Ruby.

Reply:
No, because here it's double quotes, which are processed (so `$foo` becomes the value of `foo`) and you need to escape it to get an actual dollar sign.

(I hate languages which do this difference between single and double quotes, but here we are)

Reply:
Here you can see the Ruby influence on Puppet. And I doubt that at this point you really want to change such a fundamental thing in a language.